custom/icds/templates/icds/manage_hosted_ccz.html

```diff
@@ -84,6 +85,9 @@ <h1>
                 <a data-bind="attr: { href: removeUrl}" class="btn btn-danger">
                   {% trans 'Remove' %}
                 </a>
+                <a data-bind="attr: { href: recreateUrl}" class="btn btn-warning">
```

Comment:
`btn-default`, please, we don't use `btn-warning`

Reply:
umm, i am using that for the colour it provides along with the style.
Why not use `btn-warning`?

Reply:
We don't use it anywhere else in HQ. See https://www.commcarehq.org/styleguide/molecules/#molecules-buttons

Reply:
Okay, updated. Kept both as dangerous.